introduction.md

```diff
@@ -1 +1,5 @@
-Ruby
+* Ruby
+* Python
+* HTML
+* Git
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?